advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/getting_started/installing/index.mdx

```diff
@@ -17,9 +17,9 @@ Select a link to access the applicable installation instructions:
 
 - [Oracle Linux (OL) 9](linux_x86_64/edb-dms-reader_rhel_9), [Oracle Linux (OL) 8](linux_x86_64/edb-dms-reader_rhel_8)
 
-- [Rocky Linux 9](linux_x86_64/edb-dms-reader_other_linux_9)
+- [Rocky Linux 9](linux_x86_64/edb-dms-reader_other_linux_9), [Rocky Linux 8](linux_x86_64/edb-dms-reader_other_linux_8)
 
```

Comment:
I'm curious why Rocky and Alma Linux are treated as other linux OS platforms; whereas, Oracle Linux is not?  In the end, I think the instructions for installing on any of these RHEL derivatives is the same.

Reply:
Do you mean why we have a topic to install Oracle Linux, and then a different one to install Rocky or Alma even though the command is the same? 
Not sure, maybe @nidhibhammar. @dwicinas or @josh-heyer might know.

Reply:
Sort of ... we don't really have topics for Oracle Linux 8 and 9 that are separate from the RHEL 8 and 9 topics. The Oracle Linux links point to the same pages as the RHEL links.

Reply:
Good question, I assume either the repo setup is different for RHEL/OL than from Alma/Rocky, or that maybe for other products the installation command is different. Will follow up on a Slack thread and get this PR merged since it is good to go :)